api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java

```diff
@@ -26,13 +28,16 @@
 import org.pmiops.workbench.model.ModifyParticipantCohortAnnotationRequest;
 import org.pmiops.workbench.model.WorkspaceAccessLevel;
 import org.pmiops.workbench.model.WorkspaceActiveStatus;
+import org.pmiops.workbench.monitoring.GaugeDataCollector;
+import org.pmiops.workbench.monitoring.views.MonitoringViews;
+import org.pmiops.workbench.monitoring.views.OpenCensusStatsViewInfo;
 import org.pmiops.workbench.workspaces.WorkspaceService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
-public class CohortReviewServiceImpl implements CohortReviewService {
+public class CohortReviewServiceImpl implements CohortReviewService, GaugeDataCollector {
```

Comment:
@freemabd FYI I sampled a data point from this class to get the total number of cohorts in the system. Please let me know if there's a better way to get this information periodically or if there are other metrics you'd like to see sampled.

Reply:
This service is for things related to CohortReview objects which are part of the review section of the workbench. Users can create a cohort and never use the review tool, so it doesn't seem logical to track cohort counts here. CohortReview is a subset of a Cohort. This would be nice to track, but i would use a different property for this.  `MonitoringViews.COHORT_REVIEW_COUNT`? I believe that COHORT_COUNT is a good metric to sample, but it should be sampled in CohortsController.... not sure it has a CohortsService to follow this pattern?